mailjet.cron.php

```diff
@@ -295,6 +295,10 @@ function findCode($code)
 					$content = str_replace('{'.$key.'}', $value, $content);
 				}
 
+                $content = str_replace('{shop_name}', $shop_name, $content);
+				$content = str_replace('{shop_url}', $shop_url, $content);
+				$content = str_replace('{shop_logo}', $shop_logo, $content);
+
```

Comment:
Hi,

Thanks for this new commit.
However I do not understand what you mean for the upgrade. Do you mean you won't add it or you are going to commit it later ?